### PR TITLE
The Blobbening - Removal of Roundstart Blob and the Revenge of The Outer Space Blob (Updated)

### PR DESCRIPTION
**Pull request by boy2mantwicethefam**
Attempts to make the blob more fun for both the crew and the blob. The last time the blob won by itself was in September 2023.

### Blob can't be roundstart anymore
A big source of frustrations regarding the blob were the roundstart shenanigans where it would quickly end a round because it appeared in Cargo and that cut off the crew from a steady supply of laser guns, and as most guns required Asteroid materials to be built (which couldn't be reached due to Biohazard Quarantine) it was a slow victory lap for the blob as it gradually consumed more and more of the station. In response to that the Liberator was added, which ended up over-nerfing the blob to the point it wasn't even fun for the blob player anymore, and the headache for the crew still remained because what remained was "blobstended"; as most threat points were consumed and it was a Highlander ruleset (AKA no other really dangerous ruleset could roll) it led to pretty uneventful and boring shifts.
This change disables the blob from appearing roundstart, and will, from now on, only appear at midround.

### Buffed midround blobs, a lot
Midround blobs suffered pretty badly for a long time; they came from outer space and they could barely defend their exposed tiles, they didn't enjoy the advantages roundstart blobs had of being able to position themselves anywhere they wanted, they produced even less resources, they were announced ahead of time (roundstart blobs were only announced when they appeared, whereas midround blobs are announced earlier with a "possible overmind presence" announcement) and they had to go against a prepared station rather than a station that was still setting up. Almost every single victory the blob as a whole managed to snatch (which isn't many) ever since the Liberator has been added was from the roundstart blob.
Now that the midround blob is the only blob left standing the blob can finally get away with some buffs.
- Manual expansion requires 5 chemicals instead of 15 and no longer has a 75% chance to do nothing when trying to expand into space.
- Midround blob core chemical production has been buffed from 2 per second (equal to a cerebrate, and less than a roundstart blob) to 4 per second (more than a roundstart blob by 1 point). They'll need it if they want to set a perimeter fast.
- Blob tiles now have a 50% chance to spread into space instead of 25%.
- **Update:** Blob attack damage has been split into 50% brute damage and 50% burn damage, which should allow it to deal more damage against brute-resistant crewmembers, but in exchange more suits can be used as protection against blob attacks, especially burn-resistant ones.
- **Update:** Blob attacks will now ignore about half of the total armor a target has, meaning that 50% damage reduction will count as 25% damage reduction, and 80% will count as 40%. This helps the blob not be completely shut down by well-armored crewmembers, while at the same time not making armor useless.
- Resource blobs now cost 10 points instead of 30, because the time it took for them to make a return on their investment was absurdly long (1 point every 8 seconds minimum, so in the best case scenario 240 seconds/4 minutes before they started making a net positive).
- **Update:** Liberator laser gun shots will now deal the default 25 damage to blob tiles instead of 40, and their splash damage has been converted to dealing 12.5 damage to a nearby blob tile per hit rather than do left-over damage upon destroying a tile. Notably, this means it will no longer destroy 2 blob tiles per shot, but rather destroy one and significantly damage another.

### One nerf

To compensate for what might be possible strong blob spam **the health of strong blob tiles has been reduced from 75 health to 50**, which means that 5 Liberator shots will destroy a strong blob tile instead of 8-9, 3 Antique Laser Gun shots instead of 4 shots, and 4 regular laser shots instead of 5-6. Strong blobs are super cheap to make and I think it's unfair to try to get through a sea of strong blobs that are made very cheaply.

Blob-related episode names will now happen for midround blobs.

I think the lack of unique blob skins can be addressed by counting the crew manifest and such, which I might work on later in this PR or at another date. Current idea is to unlock skins based on how many jobs there are and, for the skeleblob, if it's October. It should also allow the secblob skin to appear in secblobbed rounds.

Midround blobs are more fun to play against than roundstart blobs.

:cl:
 * rscdel: Blob: Blobs can no longer appear at roundstart.
 * rscadd: Blob: Manual blob expansion no longer has a 75% chance to fail when expanding into space, which would waste the chemicals in the process.
 * tweak: Blob: Midround blobs now produce twice as many chemicals per second (2 -> 4).
 * tweak: Blob: Manual blob expansion now costs 5 chemicals instead of 15.
 * tweak: Blob: Blobs now deal 50% brute damage and 50% burn damage when attacking. Stock up on that burn medication, and diona beware!
 * rscadd: Blob: Blobs will now ignore half of a target's armor value when calculating damage. Some of the defensive values would lead to absurd amounts of protection against the blob.
 * tweak: Blob: Strong blob tiles now have 50 health, down from 75, which means less hits are required to destroy them (3 Liberator/Antique Laser Gun shots, 4 regular laser shots, down from 4 and 5-6 respectively).
 * tweak: Blob: Blob tiles now have a 50% chance to successfully spread into space instead of 25%. Useful, as the midround blob comes from outer space.
 * tweak: Blob: Resource blobs now cost 10 points instead of 30 points, meaning they will make a return on their investment after 1 minute and 20 seconds at earliest instead of 4 minutes.
 * tweak: Blob: Liberator laser gun shots will now deal 25 damage to blobs instead of 40, which is the default for anything that isn't humanoid, and deal half of their damage to a nearby blob tile on hit, which means a Liberator shot can destroy a blob tile and damage another in one shot rather than destroy two blob tiles. Fire in a formation!
 * rscadd: Blob: Midround blobs can now influence the round's episode name.

### CODE REVIEW

**Comment by Kurfursten** [comment]
I guess I can't criticize a rework too harshly because I don't want to be the one to rework blob, but I am concerned about some of these changes.
> Cheap normal blob + no penalty for space expansion
Now unshackled from the station, the blob can make its own cult bases in space where it avoids revealing most of its mass and instead stores away huge amounts of resource blobs and cerebrates where they won't be near most players, only risking jetpack fire.

> Cheap normal blob + 2.5x faster attacks
Melee was never a good option, but now it's soul destroying. In the past you could be attacked 2 times a second, now you can be hit 5 times in a second. Each blob hit is for 30-40 brute. Meaning previously you could be hit for 60 to 80 a second at the cost of 30 chemicals. Now you can be hit for 150 to 200 a second at the cost of 25 chemicals. Was this the area the blob needed buffing...?

Personally my philosophy is totally opposite of yours, I think roundstart blobs are more interesting to fight than midround blobs because they pop in interesting locations, so it's all different. Fighting all midround blobs plays out the exact same way since they're just tumoring off a random side of the station whereas I've seen roundstart blobs in telecoms, eng secure storage, cargo, the armory, R&D, etc. I guess there's no reason to argue about this though since neither of us will convince the other.

**Comment by Eneocho** [comment]
Several of the buffs would be interesting to have as either roundstart blob or midround, but this is a bit too extreme in my eyes, and I like roundstart blob (even if a bit tiresome) for the same reasons kurf mentioned.

**Comment by Inorien** [comment]
What would you say to midround blob being a much bigger thing thats already like 50-100 blob tiles just smacking into the station all at once rather than a disadvantaged version of roundstart blob

I haven't faced many blobs recently but from memory liberators are essential, it seems like theyre cheap as chips and you can shit out a bunch and then he blob just dies. Vague memories from several years ago of (memorable) blob rounds were of desperate and drawn-out battles involving ERTs, strategically placed emitters and a militia of crew using whatever weapons they could get their hands on, rather than a five minute distraction solved by everyone rapidly getting liberators

i just get the feeling they're WAY too powerful and simple

**Comment by boy2mantwicethefam** [comment]
@Kurfursten The station is warned by the arrival of the blob and the distances involved to hide away most of its mass are too impractical unless the blob is already in a position to win the game. What truly matters is the state of the blob core and it is immutable from its location, and the mass that could possibly go into a space blob hideout is mass that does not go into the defense of the blob itself from incoming attackers. Space tiles also do not contribute to the blob's progress towards destroying the station as it only considers occupied station tiles and not space. If hiding out the cerebrates (which cannot reproduce, making the main blob core the primary threat) becomes a concern it's possible to add a means of locating them somehow.

Lower attack cooldowns were a suggestion when I asked in-game players how blob should be buffed. The cost should remain cheap because 15 points for manual spreading was horribly cost ineffective (not only could the blob buy many other things for the cost of two manual expansions but it was also a slap in the face with the 75% chance to not spread into space), but I'd be willing to remove the faster attack speed if it was replaced with armor piercing, as one particular anti-blob method involves gearing up dionae in riot armor and it's near-impossible for the blob to kill one, in-between the reduced damage and the cost of manual expansion.

Roundstart blobs are more interesting to fight but they're also a big reason why blobs are hated so much, enough for the players to be against any sort of improvement to blobs even as they lose round after round. It's possible to make a latejoiner version of the blob overmind that infects a crewmember instead much like roundstart does.

@Eneocho The buffs may seem extreme but midround blobs are severely disadvantaged compared to roundstart blobs, and I still predict that they will fall to massed Liberator fire, Cargo guns and R&D guns.

@Inorien The midround blob already appears with a few other blob nodes scattered about due to the blob meteor storm.
I intend to nerf Liberators if the PR is merged as part of the whole "balancing blob" thing, I don't believe they should deal as much damage to the blob as the captain's antique laser gun at the very least.

**Comment by SonixApache** [comment]
compress :cl:, preface all with Blob:

**Comment by hacker-on-steroids** [comment]
make a tier 2 strong blob which is airtight
make blob evoluate faster with oxygen present

i think liberators could also be toned down(or made automatic or something, having to spend 5-30 minutes spamming left click is just insane)
cheaper resource blobs are the only thing i am in favor of

**Comment by boy2mantwicethefam** [comment]
@SonixApache I like being descriptive with the changelogs but I will preface them with Blob:

@hacker-on-steroids If the blob comes from outer space it will breach and suck the air out.

**Comment by Eneocho** [comment]
> If the blob comes from outer space it will breach and suck the air out.

Clearly the solution is for the blob to place airtight blobs (I think they do already and can toggle air tightness) on all breached tiles so they can still evoluate faster with oxygen.

**Comment by boy2mantwicethefam** [comment]
That's one other problem, how should it work out? Should it matter whether the blob core has access to fresh air or just any blob tile? Should the oxygen be consumed by the blob? Should it be represented by extra chemical production or faster automatic blob generation? Overall it becomes out-of-scope for the current PR, and I am not sure if the current PR will pass in the first place in order to be able to work with blobs more.
I think strong blobs were already supposed to be air-tight, or maybe that's just a thing other servers did, but they are immune to fire.
They can already toggle whether the blobs automatically destroy surrounding walls when expanding.

**Comment by Kurfursten** [comment]
> dionae in riot armor and it's near-impossible for the blob to kill one

When are we going to accept diona are the problem? When an entire race is gimmicked around being invulnerable to brute and basically all simple mobs only work in brute, that's a problem. Parse what you're proposing as "we should be making the blob hit every crewman for 3x more damage so that it can engage with a really stupid race", it's slapping another bandaid on a really bad design.

**Comment by boy2mantwicethefam** [comment]
Countering the Diona strategies could be easy by making half of the blob's damage deal burn damage.
I will put the delay back but the blob should ignore about 50% of armor on each hit so that things like riot armor or the miner's hardsuit do not convert 30-40 damage into 6-8 damage per hit (80% damage reduction). The cost should remain as is because 15 points is still obscenely awful.

**Comment by DeityLink** [comment]
Roundstart blob is iconic, you can't just take it away! On top of that Blobs are more interesting for the crew to fight inside the station than outside for various reasons.

**Comment by jwhitak** [comment]
Double buff to damage? Split damage type AND ignores half resistance? Faster expanding? Not a Highlander ruleset (any antag whose goal is to end the round is a highlander)? No more round start blorbs? Nerfing strong blob?

But not hitting liberators?

What did I walk into?

**Comment by boy2mantwicethefam** [comment]
@DeityLink Something's gotta be done or else blob will remain in this purgatory of "everyone hates it and no one actually wants to do anything about it" for quite a long time, remember how people downvoted [a change that prevented the core from being picked up by the subspace tunneler](https://github.com/vgstation-coders/vgstation13/pull/35120) and it took going against the emojicracy to merge it. I mean I don't hate blob but playing blob has been a miserable experience for more than a year now, it really hasn't won a round properly since 5 months ago.
I plan to add a latejoin variant of the blob if the PR gets merged, one where a new arrival has the option to agree to become a blob host.
@jwhitak I intended to leave any Liberator nerf to a subsequent PR, but I should probably set their blob damage to 25 and get rid of the splash damage, now that the blob is midround and the station would have enough time to arm itself most of the time. Liberators should be a cheap, lower-risk weapon that is distributed to the crew during an emergency and not The Blob Melter 9001, with comparatively low damage but with a good amount of shots to fend something off and the ability to recharge much faster. This should complement the strong blob nerf, letting it destroy a strong blob in 5 shots with this change, encouraging firing lines instead of "one guy and 5 Liberators in his bag of holding". Or they could try getting better guns, that's always an option with midround blobs.
It cannot attack faster anymore but its damage type has been made varied, which should help it fight against dionae.
The reason it shouldn't be a Highlander ruleset anymore is that a roundstart Highlander ruleset has already likely done its damage if the round didn't end right then and there, and a blob overmind should spice up the round if Dynamic can even roll it in the first place.

**Comment by Kurfursten** [comment]
Switched stance from neutral to against as a result of removing the unique feature of the liberator

**Comment by ErrorsWindows** [comment]
who even made sneederators you cant even load them with .357

**Comment by boy2mantwicethefam** [comment]
I can try adding the splash damage back to the Liberator but in such a way that it deals half of its damage to a nearby blob tile on hit.

**Comment by jwhitak** [comment]
> I intended to leave any Liberator nerf to a subsequent PR

That's the only change necessary here for Blob. Liberators are too strong. That's it. Old Blob you had to choose between Cargo and Engineering (with science as a distant third) as they were the only primary threats to you (or be a chad, choose neither, and try to take advantage of your superior positioning to win). This is true even of lategame blob, though science actually may be a threat at this point in the game. They have their research after all! Now, as of current game, you have to pick one between Cargo, Engineering, and Security. And the other two will ruin you. At roundstart. Midround? Those three plus science. You need to lessen the impact of one of the deparments here. And the easy answer? Nerf liberators.

And you did! Now that blob won't get utterly screwed by security blobdestroyer9000s, they might actually stand a chance.

> The reason it shouldn't be a Highlander ruleset anymore is that a roundstart Highlander ruleset has already likely done its damage if the round didn't end right then and there, and a blob overmind should spice up the round if Dynamic can even roll it in the first place.

wat

Highlander's goal is to end the round regardless of any other antags, and if they succeed, the round ends. That's the definition. Unless you remove the ability for a midround blob to end the round, it's literally a highlander ruleset, Your complaint seems to rest more with the ability for only one highlander ruleset to be fired at once. Having a mid round blob in the *middle of nuke ops* does not sound like a fun round to me. After they've been defeated? Sure. Make a PR to let secondary highlanders fire if the first are defeated.

Aside from those direct quotes, these are some further thoughts:

* force blobs midround

This is a pro and a con. Roundstart blob warps the round so that "it's a blob round!" and people will just start mass suiciding because nobody wants to rebuild. I've seen rounds with 6+ doctors who all suicide or go braindead when the injured start coming in. So, I can see advantages in forcing blob to wait a little while before popping off. Get people invested in the round. The con? Blob is REALLY BAD OFF in a midround position. Cargo will have finished their orders, engineers will have set up power, and science will have done their research. Everyone will be ready and waiting to screw you up, unless you're able to do some preptime to stop it early on, or hit them fast enough so that they're not ready. Or, fundamentally rework Blob so that they can handle a prepared crew fairly. Which this PR is attempting to do. Hm.

* split 50/50 brute/burn, ignores 50% armor

Notably, melee resistance is used in all cases here, which is 80% at best, reduced by 50% due to the buff. A normal human in a riot suit will take at least 9 brute and 9 burn on a hit, up to 13/13 depending on RNG, taken as two separate blows. (Without armor, that's 15/15 to 20/20. Additionally, the old numbers were min 6 brute against a riot suit human, max 8, unarmored 30-40.) While this looks like a considerable buff on paper, in practice it's more complicated. There will be a lot less broken bones and instantly exploded limbs, but burn paincrits will be more common. Diona in a riot suit will be dumped into crit in about three hits. No armor? Twoish hits.

My takeaway is that melee will be considerably more difficult. Can't just grab bicaridine and riot armor anymore. And Diona are now straight up a mistake against it due to their considerable burn weakness.

* General cost improvements

Looking at the changes, these will likely give a midround blob a chance to survive a fully armed station.

# TL DR

yeah sure merge this... there are interesting changes we can stand to see with blob going on here. but personally: don't remove highlander. and consider rewriting it so that roundstart blobs are a thing again, but take some time to pop (so they can possibly sabotage while letting the round grow naturally).

**Comment by boy2mantwicethefam** [comment]
> Liberator nerfs

I would have left it at that but any change that ends up buffing the blob ends up stonewalled by players that really hate fighting blobs, partly out of experience and partly as a meme that is taken seriously. There were 2 attempted nerfs against the Liberator (one simply reverted a buff) and they both failed. Since the hate seems to stem from fighting roundstart blobs I figured it would be easier to attempt to do away with roundstart blobs and then work from there in balancing the blob, since it's also very difficult to balance the blob when the two states it can appear in (roundstart and midround) are so vastly different in power, with roundstart blobs having massive advantages over midround blobs, and midround blobs certainly need some buffs. The Liberator has the potential to be a better weapon than just "hey print this en masse to kill the blob and then forget about it until the next round a blob appears" (I don't believe I have ever seen it used in a non-blob round) but I'm not sure what niche it could fulfill; miners already have their own gun and the Warden has armory access. Science could have one on the outpost but it could lead to easier lockbox-breaking shenanigans. Maybe as part of weapon vendor merchandise on Castle? The name "Liberator" would sound cool on Castle.

> Highlander

I did it with the idea that there was no real harm in having it as a non-highlander ruleset due to what sort of situations it could appear in, more specifically that most roundstart highlander rounds end early and if they didn't end early then the station is likely capable of fighting a midround blob. However, I do recognize that the highlander ruleset is meant mainly for exceedingly dangerous antagonists that can end the round early by fulfilling their special conditions, so I will put it back.

I want to look into allowing melee weapons to do full damage on hit if they are sharp enough in a future PR.

**Comment by falcon2346** [comment]
Mixed feelings on this. I really dislike blob, It already demands the attention of the entire station to effectively take care of it so there isn't much room for rp or fun gimmicks. Even after they're dead they cause massive station damage, destroying all the pipes, wires, and walls around them, often forcing a shuttle call. Any buff to them makes me recoil in disgust.

However, a lot of that is specifically a round start blob problem. A midround blob means you have a bit of breathing room to play the game before the blob demands your attention, and since they start in space they're less likely to destroy essential pipes and wires.

>I'm not sure what niche it could fulfill

Liberators are best used for busting lockers outside of blob rounds.

**Comment by BomberBro** [comment]
Blob is extremely dependent on crew competence to win/lose. If there's a few engineers and cargo/science is sentient then they'll win, otherwise the blob'll get btfo, moreso than any other antag in the game. I'd wager a lot of blob losses are due to noobs not knowing all the alt/shiftclicks or that strong blobs rape lasers and block gasses, it's an antag that people only rarely get.

**Comment by boy2mantwicethefam** [comment]
I've witnessed blob rounds multiple times, playing both as and against blobs, enough times to rule out inexperience as being the main cause of blob losses, they tend to spam strong blobs, put up a good defense, arrange special nodes in a relatively optimal way, and even start creating cerebrates but eventually enough Liberator fire gets massed that the blob cannot withstand the sheer amount of damage and then it gets destroyed after a while. On one occasion I killed a midround blob myself in just two trips (returning once to recharge the guns), carrying Liberators in a bag of holding and shooting at it from space.
I mentioned that the last time a blob has naturally won was in September 2023, after ruling out special antag-flooded rounds like Antag Madness. It's not an exaggeration, the blob really hasn't won in 5 months. If blobs were consistently losing due to no player skill then roundstart blobs wouldn't have been loathed so much by a part of the playerbase, but what happened is that a weapon that is extremely effective against it was added and the blob remained disliked even as it started losing most rounds. Most of the dislike stems from roundstart blobs specifically, which the PR removes in favor of buffing the far weaker (compared to roundstart) midround blobs.

**Comment by Theophrastus1** [comment]
As a blob enthusiast, liberators are the main issue with blobs. Prior to liberators if the station focused fire correctly or engineering got emitters set up properly, that was an issue all on its own and you had a good shot at losing, but now the fact that they can just shit out liberators from the very second you pop is insanely difficult and just means you're going to lose unless some retard carries every single one and you kill him.

I think liberators are good to specifically fight the blob, but i also think they are far too good at what they do and essentially trivialize it for the player side.

**Comment by ghost** [comment]
@boy2mantwicethefam @Kurfursten 
Would you mind me making an alternative PR that nerfs Liberators instead? It looks like people don't want the roundstart blob to go.
>Increase plastic cost: yes
>Reduce shots back to 6: yes
>Increase liberator size: yes
>Increase charge time: dependent on how much it was changed

**Comment by Kurfursten** [comment]
Go nuts

**Comment by boy2mantwicethefam** [comment]
@github-user208604 I wanted to make a more thorough blob mini-rework because the blob is stuck in a dilemma where the blob can't really be changed (emojicracy vetoes any attempt at buffing the blob due to how strong roundstart blobs are) and it's a miserable experience to play all around for both the crew (station remains a mess and enters "blobstended" if it's roundstart) and the blob (wiped in a short time due to Liberators).
- Roundstart blobs are extremely strong and won very often before Liberators appeared, after which they rarely ever won, but it's still two extremes of "roundstart blob ends the round almost all the time" and "roundstart blob gets wiped almost all the time"
- Midround blobs have a myriad of disadvantages and also have to contend with a better-prepared station, and they rarely win unless the station is too messed up to mount a defense


Nerfing Liberators in those styles won't fix the issue of how much they tear through a blob and how they can be massed unless the plastic cost is excessive enough (dozens of sheets each) but at that point the cost ends up unrealistic because there's other way cheaper printable guns. Alternatives were suggested such as locking Liberators behind research but that still means moving back to the "roundstart blob auto-wins" scenario while midround blobs remain completely disadvantaged. I won't mind changing the Liberator but I don't think it will fix the aforementioned issues.

**Comment by ghost** [comment]
@boy2mantwicethefam Fair enough, but I think it would make more sense to take it one step at a time and nerf Liberators first to see if the blob starts winning again, then do all the work overhauling the blob itself.
Like you yourself mentioned:
>     * Roundstart blobs are extremely strong and won very often before Liberators appeared, after which they rarely ever won, but it's still two extremes of "roundstart blob ends the round almost all the time" and "roundstart blob gets wiped almost all the time"

Wouldn't it make more sense to nerf Liberators first by that logic? IMHO overhauling blob seems like a waste of time when there's a much simpler and more obvious solution handled to you on a silver plate in that very PR where you tried to take things in the right direction.
Have you never heard of Occam's razor?

**Comment by boy2mantwicethefam** [comment]
There were two attempts at nerfing Liberators (one was reverting the change that gave them 10 shots back to 6, and the other was reducing their damage) and neither of them passed due to people preferring them to remain blob deleters because roundstart blobs are extremely powerful. I would have liked doing it one step at a time but few people are willing to make a concession and small changes tend to be far more hotly debated than system-changing stuff for some reason, it is easier to simultaneously appease both groups with one bundle of changes that tries to solve a wide range of concerns rather than performing each change individually, if the changes were just blob buffs or just roundstart blob removal I believe either would have ended up with a far worse reception than the current PR. Without this bigger change nothing would get really done because nobody wants to say "let him cook", and blob would remain a miserable experience for both the crew and the blob.
I plan to look more at Liberators if the PR is merged because I still believe they would stomp midround blobs and I'll need more rounds to look at. I plan to shift their role towards something more of an "anti-xenofauna" tool rather than "anti-blob" specifically, because as it is the only reason people use it in any round is because there's a blob despite its ability to deal extra damage to creatures.
I am aware of Occam's Razor even if it pertains more to "the most straightforward explanation is probably the best" so I'll shorten it all to "people don't want any individual buffs or nerfs to blobs and the only way to get anything done is to combine both, and currently everyone hates playing a blob round, even the blob player"

**Comment by KingofHONK** [comment]
I mean no matter which way you slice it, if the last blob victory really was almost 5 months ago that's pretty _fucking_ grim. I think even your most staunch anti blobbers can't defend that.  I recall crews winning quite often in the pre-liberator days too, so it's a reasonable choice to buff blobs.

**Comment by ghost** [comment]
Yeah, that's convincing enough. There's no reason not to test merge this.

It would be great to see roundstart blobs make a comeback in the future though.

**Comment by Kurfursten** [comment]
1. I disagree with most of the changes here - particularly related to weakblobs
2. I think trading a fun, always-different game mode (roundstart blob) that's boring and always plays out the same (midround blob) just because it's easier to balance is a disastrous mistake
3. Roundstart blob is also the original mode and we're losing it
4. Vote is pretty mixed

In most circumstances I would just kill this because I disagree with it and there isn't enough overwhelming support to sway me, but no one has touched blob in a while. I personally think barryblob will be a cursed era, but we can try it.

**Comment by angryturnip** [comment]
Skeleblob in October and clown blob in April would both be great uses of the skins by the way

**Comment by gurfan** [comment]
mistake
